src/blockdata/script.rs

```diff
@@ -524,7 +524,7 @@ impl Script {
         // special meaning. The value of the first push is called the "version byte". The following
         // byte vector pushed is called the "witness program".
         let script_len = self.0.len();
-        if script_len < 4 || script_len > 42 {
+        if !(4..=42).contains(&script_len) {
```

Comment:
FTR, this feels weird because it looks like Yoda condition. Hypothetical `!script_len.is_in_range(4..=42)` would read better.

Reply:
Noted, thanks. I think its better than the original, will change if others object also.